docs/pages/IconButton.md

```diff
@@ -1,30 +1,43 @@
 # IconButton
 
-The `IconButton` component is a `<button>` element with icon.
+The `IconButton` component is a transparent `<button>` element with an icon.
 
 ```.jsx
 <IconButton
-  name='Flame'
-  size={24}
-  color='orange'
-  title='Set on fire'
-  onClick={() => {}}
+  title='Search button'
+  onClick={() => {console.log('Search button clicked!')}}
+  mr={2}
+  icon={
+    <Search
+      color='primary'
+      size={36}
+    />
+  }
+/>
+<IconButton
+  disabled={true}
```

Comment:
```suggestion
  disabled
```

Reply:
No problem! 😄 Thanks for catching that, fixed!